### DISCUSSION TITLE
How to load stdlib via a script tag:

### DISCUSSION THREAD

**Discussion by orenbh1** [stdlib-js/stdlib#1282] [General]
1. You may add a script tag to a specific function:
`<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/stdlib-js/stats-base-dists-poisson-pmf@umd/browser.js"></script>`
Example: pmf(3990,4000)

2. You may add a script tag to a specific namespace:
For example, to load all Poisson APIs, see [@stdlib/stats-base-dists-poisson](https://github.com/stdlib-js/stats-base-dists)
Example: poisson.cdf(3990,4000)

To load all "base" distributions, see [@stdlib/stats-base-dists](https://github.com/stdlib-js/stats-base-dists)

3. You may add a script tag to a specific top-level namespace:
For example load all the statistics functions: [@stdlib/stat](https://github.com/stdlib-js/stats)

Thank you Athan!!! (@kgryte)

**Comment by kgryte** [comment]
@orenbh1 Thanks for posting. You're right. We should add something like this to the main README as part of the user story section.
